internal/provider/file/file_test.go

```diff
@@ -153,13 +157,35 @@ var (
 func TestListJobFilename(t *testing.T) {
 	fc := file.New(&model.PrdFile{
 		Filename: "./fixtures/dockerhub.yml",
-	})
-	assert.Equal(t, dockerhubFile, fc.ListJob())
+	}, defaultImageDefaults)
+
+	expectedFiles := dockerhubFile
+	actualFiles := fc.ListJob()
+	assert.Equal(t, len(expectedFiles), len(actualFiles))
+	for i, expected := range expectedFiles {
+		assert.EqualExportedValues(t, expected, actualFiles[i])
+	}
 }
 
 func TestListJobDirectory(t *testing.T) {
 	fc := file.New(&model.PrdFile{
 		Directory: "./fixtures",
-	})
-	assert.Equal(t, append(append(bintrayFile, dockerhubFile...), append(lscrFile, quayFile...)...), fc.ListJob())
+	}, defaultImageDefaults)
```

Comment:
Same as previous comment

```suggestion
	}, &defaultImageDefaults)
```